src/core/server/services/stories/index.ts

```diff
@@ -94,6 +94,8 @@ export async function findOrCreate(
     StoryCreatedCoralEvent.publish(broker, {
       storyID: story.id,
       storyURL: story.url,
+      tenantID: tenant.id,
```

Comment:
I want to move the `tenantID` out of the publish call, I'll be submitting a PR shortly that will add the tenantID to the payload instead at a higher level.

Reply:
Note that this is now available on the event object itself (not the `data` member).

Reply:
Great! I already removed it from there.